examples/blueprint-construct/index.ts

```diff
@@ -65,14 +66,16 @@ export default class BlueprintConstruct extends cdk.Construct {
             new ssp.addons.VpcCniAddOn(),
             new ssp.addons.CoreDnsAddOn(),
             new ssp.addons.KubeProxyAddOn(),
-            new ssp.addons.OpaGatekeeperAddOn()
+            new ssp.addons.OpaGatekeeperAddOn(),
+            new ssp.addons.KarpenterAddOn(),
         ];
 
         const blueprintID = `${blueprintProps.id}-dev`;
 
+        tagSubnets(this,blueprintProps.vpc.privateSubnets,`kubernetes.io/cluster/${blueprintID}`,"1")
```

Comment:
If this is a prereq for the add-on to work, then this tagging should occur in the add-on body. You should be able to get the VPC and subnets, if not let's review what support we need to add for it.

Reply:
It should be doable. I was going back and forth based on the fact that the examples created a separate VPC stack. Will address this in the next commit.

Reply:
Initial commit addressed this by calling a function that leverages `cdk.Tags` - this only manages tags of resources created within the scope so any VPC subnets created outside the stack fails. Implemented a custom resource in case users provides a pre-created VPC to the blueprint. 

Addressed in the latest commit.